style.css

```diff
@@ -0,0 +1,184 @@
+html {
+  color: #222;
+  font-size: 1em;
+  line-height: 1.4;
+}
+
+::-moz-selection {
+  background: #b3d4fc;
+  text-shadow: none;
+}
+
+::selection {
+  background: #b3d4fc;
+  text-shadow: none;
+}
+
+hr {
+  display: block;
+  height: 1px;
+  border: 0;
+  border-top: 1px solid #ccc;
+  margin: 1em 0;
+  padding: 0;
+}
+
+audio {
+  vertical-align: middle;
```

Comment:
You should phrase the comments politely instead of ordering tone.

Instead of
```
You have used the vertical-align: middle declaration multiple times in your CSS. Instead, you can create a class.
```
You can say
```
It will be better if you use a class for ` vertical-align ` instead of setting this property for multiple items.
```